src/System.IO/tests/BufferedStream/BufferedStream.InvalidParameters.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class BufferedStream_InvalidParameters
+    {
+        [Fact]
+        public static void NullConstructor_Throws_ArgumentNullException()
+        {
+            Assert.Throws<ArgumentNullException>(() => new BufferedStream(null));
```

Comment:
Throughout these tests when using Assert.Throws for argument exceptions, wherever possible it'd be good to use the overloads that take a first string parameter that let you specify the name of the parameter.  Then Throws will verify that the exception includes the right parameter name.